docs/developer-docs/integrations/sns/tokenomics/preparation.md

```diff
@@ -8,44 +8,30 @@ for examples using the
 [SNS tokenomics training & tools](https://wiki.internetcomputer.org/wiki/How-To:_SNS_tokenomics_configuration)
 as described in [this section](./tokenomics-intro.md).
 After deciding on these configurations, the next step is to
-translate them into 
-[initial parameters](../launching/launch-steps.md/#SNS-launch-step-preparation) <!-- NEW: UPDATE LINK --> that the SNS will be initialized with during the SNS launch.
+translate them into initial parameters that the SNS will be initialized 
+as the [first Step of the SNS launch process](../launching/launch-steps-1proposal.md).
 
 
 This section explains on a technical, but high level, how initial parameters are set and how they affect different canisters.
 The [SNS CLI tool](https://github.com/dfinity/ic/tree/master/rs/sns/cli)
 allows you to set these parameters and includes detailed descriptions of all parameters.
 
 ### Initial parameters
-<!-- NEW: All SNS parameters are set in a **.yaml** file that can then be passed as an argument to the NNS proposal that triggers the SNS launch, both for testing and in production.
-There are some parameters that have to be set and others that are set to a default value but that can also be changed. To make sure that all parameters are set to valid values, that are also consistent
-with each other, there is a tool to validate a given input file. -->
-Apart from the decentralization swap parameters
-(see next section), all SNS parameters are set in a **.yaml** file that can 
-then be passed as an argument
-when installing the SNS, both for testing and in production.
-There are some parameters that have to be set and others that are set to a
-default value but that can also be changed.
-To make sure that all parameters are set to valid values, that are also consistent
-with each other, there is a tool to validate this input file. 
+All SNS parameters are set in a **.yaml** file that can then be passed as an argument to the NNS proposal that triggers the SNS launch, both for testing and in production.
+There are some parameters that have to be set and others that are set to a default value but that can also be changed. To make sure that all parameters are set to valid values, that are also consistent with each other, there is a tool to validate a given input file.
 
 **To create the SNS parameter yaml file and validate it, follow the steps
 ['Creating the configuration file'](https://github.com/dfinity/ic/tree/master/rs/sns/cli#creating-the-configuration-file)
 in the SNS CLI tool.
-The tool is also available through the dfx cache like this: `$(dfx cache show)/sns`**
+The tool is also available through the dfx cache like this: `$(dfx cache show)/sns` **
```

Comment:
This works too, but we should encourage them to start using dfx extensions instead. 

Let me ask if there is a doc section for the sdk on this tool yet.

Reply:
That would be great. Could you then maybe add a proposl for what to say here: i.e. how to devs get / create this file?

Reply:
let's remove this and the bullet point above and just link to the template / example files

Reply:
linked to the two yaml files and closing this.